src/components/breadcrumbs/__snapshots__/breadcrumbs.spec.tsx.snap

```diff
@@ -0,0 +1,40 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
```

Comment:
You need to update the snapshots when you run the tests, this should have been deleted

`npm test -- -u`